docs/sdk-and-tools/elastic-search.md

```diff
@@ -67,6 +67,47 @@ In case of a single machine, our recommendation is as follows:
 
 ## Clone an Elasticsearch cluster
 
-In order to have all the information about the Elrond chain in an Elasticsearch cluster (from genessis to current time) one has to copy all the data with a specific tool from an Elasticsearch cluster to another.
+In order to have all the information about the Elrond chain in an Elasticsearch cluster (from genesis to current time) one has to copy all the data with a specific tool from an Elasticsearch cluster to another.
 To get more information how to do this use the documentation from this [repository](https://github.com/ElrondNetwork/elrond-tools-go/tree/main/elasticreindexer).
 
+
+## Elasticsearch indices 
+
+An observing-squad with the elastic indexer enabled will save data in different indices. This data is used to solve a lot of use cases. An example is to fetch all the 
+transactions that belong to an address or to display all the address sorted based on the EGLD balances.
+Saved data is grouped into indices.
```

Comment:
You can delete it, since you've already mentioned it before : 
>An observing-squad with the elastic indexer enabled will save data in different indices

```suggestion
```

Reply:
done